.commitlintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+    "extends": [
+        "@commitlint/config-conventional"
+    ],
+    "rules": {
+        "type-enum": [
+            2,
+            "always",
+            [
```

Comment:
Is there a way to force `prefix: text` without defining a limited set of prefixes?

Or -- what is the minimal configuration you feel we could start with here that provides some basic value?